dz-9/src/main/java/org/example/Person.java

```diff
@@ -0,0 +1,53 @@
+package org.example;
+public class Person {
+    private String firstName;
+    private String lastName;
+    private int age;
+    private String gender;
+    private int pensionAge;
```

Comment:
Add field `Person partner` and set it in `registerPartnership ` and `deregisterPartnership`